### PR TITLE
fix(core) Remove ngMaterial from server view

### PR DESCRIPTION
**Pull request by trainerbill**

### CODE REVIEW

File: modules/core/server/views/layout.server.view.html [hunk 1]
```diff
@@ -18,17 +18,14 @@
...
-    </md-toolbar>
-    <div layout="row" flex>
-      <md-sidenav class="md-sidenav-left md-whiteframe-z2" md-component-id="coreLeftNav" ui-view="leftnav"></md-sidenav>
-      <md-content flex ui-view="main"></md-content>
-      <md-sidenav class="md-sidenav-right md-whiteframe-z2" md-component-id="coreRightNav" ui-view="rightnav"></md-sidenav>
-    </div>
-    <footer ui-view="footer"></footer>
-  </div>
+  <header ui-view="header"></header>
+  <section>
```
Comment (siteshackinc) [inline modules/core/server/views/layout.server.view.html]: section layout="row"

File: modules/core/server/views/layout.server.view.html [hunk 2]
```diff
@@ -18,17 +18,14 @@
...
+    <section ui-view="leftnav"></section>
+    <section ui-view="main"></section>
```
Comment (siteshackinc) [inline modules/core/server/views/layout.server.view.html:24]: maybe

section layout=column
  section ui-view="main"
  section ui-view="footer"
/section

and remove footer ui-view="footer"

Comment (trainerbill) [inline modules/core/server/views/layout.server.view.html:24]: layout=column is ngMaterial.  want to keep it out if possible

File: modules/core/server/views/layout.server.view.html (hunk 1 above)
Comment (trainerbill) [inline modules/core/server/views/layout.server.view.html]: why is it needed?  that is ngMaterial.  I am not seeing any detrimental side effects by not having it

**Comment by trainerbill** [comment]
@siteshackinc You good with this?

**Pull request merged**